packages/client/utils/draftjs/convertToTaskContent.ts

```diff
@@ -1,11 +1,17 @@
 import {ContentState, convertToRaw} from 'draft-js'
 import entitizeText from './entitizeText'
+import isAndroid from './isAndroid'
 
-export const removeSpaces = (str: string) =>
-  str
-    .split(/\s/)
-    .filter((s) => s.length)
-    .join(' ')
+export const removeSpaces = (str: string) => {
```

Comment:
@gitstart-parabol I looked a little deeper into the problem. I see that we already check for `isAndroid` on higher lever here
https://github.com/ParabolInc/parabol/blob/78e0a8386a325cac3bcdb30608b271eec955e304/packages/client/components/DiscussionThreadInput.tsx#L225

And we do not use `convertToTaskContent` function on non-android devices, which means we do not strip any spaces at all.

It also says `convert to task` but there is no `task`, which is a bit weird.

We probably don't need to fix `removeSpaces` function.

What if we do the following:

Replace 

https://github.com/ParabolInc/parabol/blob/78e0a8386a325cac3bcdb30608b271eec955e304/packages/client/components/DiscussionThreadInput.tsx#L229

With 

https://github.com/ParabolInc/parabol/blob/0dbd37ee1cefaeeee27a11cf547cf710f321a7fd/packages/client/utils/draftjs/convertToTaskContent.ts#L25-L26

Would something like this work?

Reply:
@igorlesnenko, Thank you for making the suggestions; they were really helpful in resolving the problems.